x-pack/plugins/discover_enhanced/ui_tests/README.md

```diff
@@ -0,0 +1,17 @@
+## How to run tests
+First start the servers with
+
+```bash
+// ESS
+node scripts/scout_start_servers.js --stateful
+// Serverless
+node scripts/scout_start_servers.js --serverless=es
+```
+
+Then you can run the tests multiple times in another terminal with:
+
+```bash
+npx playwright test --config x-pack/plugins/discover_enhanced/ui_tests/playwright.config.ts
```

Comment:
Thanks for not wrapping this in a script 🙏🏼

Reply:
No wrapping because we want you to be able run the tests using all Playwright Test capabilities (cli, IDEA plugins, etc.)